docs/design/disqualification.md

```diff
@@ -0,0 +1,77 @@
+# Node Disqualification
+
+## Abstract
+
+This design doc outlines how we will implement node disqualification, wherein a storage node is deemed permanently inelligible for storing new data via uploads.
+
+Nodes can be disqualified due to audit failures, failures to return data during repair, or poor uptime statistics.
+
+## Main Objectives
+
+1. To not send new data to nodes which are not behaving as desired.
+2. Efficiently identify and ignore these 'bad' nodes during various node selection and statistics processes.
```

Comment:
Another objective would be to improve object durability by having the repair checker consider not only offline nodes, but also online, but disqualified nodes.

Reply:
"various node selections" includes the repair checker query.

Reply:
Repair checker currently does care about 'unreliable' nodes at the moment... 'unreliable' was the term we created for 'we aren't doing disqualification, but somehow we're doing this other thing'.